edv/test_scripts/client/testlists/testlist_efispec.sh

```diff
@@ -1,66 +1,58 @@
 #!/bin/sh
 
+# Get the common routines to parse command line
+source ${RUNDIR}/testlists/testlist_common.sh
+
+# Get the global environment variables
+source ${RUNDIR}/scripts/client_env.sh
+
+# Parse the command line
+parseAndSetParameters $@
+
 export SCM="700G" #700G
 export NVME="30T" #1T
 
 export SRV_HEADNODE="edaos09"
 export MPI="IMPI"
-export RUNDIR="/panfs/users/schan15/client"
-export BUILDDIR="/panfs/users/schan15/builds"
-export TB="daos_rel201rc1"
+export RUNDIR="/panfs/users/${USER}/client"
+export BUILDDIR="/panfs/users/${USER}/builds"
 export MOUNTDIR="/tmp/daos"
-export APPSRC="/panfs/users/schan15/apps/${MPI}/efi_johann"
+export APPSRC="/panfs/users/${USER}/apps/${MPI}/efi_johann"
 export APPNAME="efispec"
 export APPRUNDIR="${MOUNTDIR}/efi_johann/test"
 export OUTDIR="${MOUNTDIR}/efi_johann/test"
 export APPINFILEDEST="${APPRUNDIR}"
-export APPINFILE="/panfs/users/schan15/client/input/e2vp2.cfg.big"
+export APPINFILE="/panfs/users/${USER}/client/input/e2vp2.cfg.big"
```

Comment:
We should try to store this input files under the git repo... Otherwise, the scripts may not work.

Reply:
Very good comment. I agree.